src/components/Tooltip/TooltipPlacement.ts

```diff
@@ -7,3 +9,17 @@ export type TooltipPlacement =
     | 'top-right'
     | 'center-left'
     | 'center-right';
+
+const tooltipPlacementMap = {
+    'bottom-left': 'bottom-start',
```

Comment:
Similar to how types have been defined for `TooltipPlacement`, can we also define them for `PopperPlacement` _(is that the same as `Placement`?)_ and take advantage of that to type `tooltipPlacementMap`?

Reply:
I am not sure why are we trying to keep original placement values and try to map them to @popperjs Placement type values, when we could just directly use Placement values from @popperjs/core/lib/enums?

Reply:
We are keeping them because otherwise we will introduce breaking changes. The idea is to communicate that those value are deprecated, but we can't remove them now.